.github/workflows/pkg.yml

```diff
@@ -36,7 +39,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.18
+          go-version: 1.19
```

Comment:
```suggestion
          go-version-file: 'go.mod'
```

Reply:
I noticed that below here we are running `build` and `test` with verbose logging - might get some speedup from dropping the `-v`.